src/CoreShop/Component/Index/Getter/FieldCollectionGetter.php

```diff
@@ -44,9 +51,30 @@ public function get(IndexableInterface $object, IndexColumnInterface $config): m
         }
 
         foreach ($validItems as $item) {
-            if (method_exists($item, $fieldGetter)) {
-                $fieldValues[] = $item->$fieldGetter();
+            $fd = $item->getDefinition()->getFieldDefinition($config->getObjectKey());
+
+            if (!$fd) {
+                continue;
+            }
+
+            if (!method_exists($item, $fieldGetter)) {
+                continue;
             }
+
+            if ($fd instanceof Localizedfield) {
```

Comment:
@dpfaffenbauer I think it would work but $fd is not instance of LocalizedField. It is instance of field under the localizedfields. Thats why I used
`                if ($item->getDefinition()->getFieldDefinition('localizedfields')) {
`

Reply:
but that only checks if there are localized-fields in the fieldcollection... which is useless basically. We need to know if the selected field is localized or not. But I see that there is an issue here. I will check again.

Reply:
For some reason that checks if that field is inside localized-fields. If you have input and localized input that will be true only for localized input. Not sure why is that the case tho.

Reply:
changed some more, worked for me now.